Makefile

```diff
@@ -11,7 +11,7 @@ endif
 SHELL := /bin/bash
 ORG := github.com/operator-framework
 PKG   := $(ORG)/operator-lifecycle-manager
-MOD_FLAGS := $(shell (go version | grep -q -E "1\.1[1-9]") && echo -mod=vendor)
+MOD_FLAGS := $(shell (go version | grep -q -E "1\.1[1-4]") && echo -mod=vendor)
```

Comment:
Any idea on why this was needed?

Reply:
I've since reverted it. But from the notes, you don't need -mod=vendor from 1.14 I think.